xtask/src/main.rs

```diff
@@ -236,12 +556,14 @@ pub mod tasks {
             for query_file in query_files {
                 let language = get_language(&grammar_name);
                 let query_text = read_query(&language_name, query_file);
-                if !query_text.is_empty() && language.is_ok() {
-                    if let Err(reason) = Query::new(language.unwrap(), &query_text) {
-                        return Err(format!(
-                            "Failed to parse {} queries for {}: {}",
-                            query_file, language_name, reason
-                        ));
+                if !query_text.is_empty() {
+                    if let Ok(language) = language {
```

Comment:
I also did it here 😅 https://github.com/helix-editor/helix/pull/3690/files#diff-9c5e61698d6d8a8ef4a3da7075c31857769134a932ac141d4eb62f68f131aa35R243 mine is shorter if you wan to take it